app/src/main/java/org/mifos/mobilebanking/ui/activities/SplashActivity.java

```diff
@@ -0,0 +1,20 @@
+package org.mifos.mobilebanking.ui.activities;
+
+/*
+ * Created by saksham on 01/June/2018
+ */
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import org.mifos.mobilebanking.ui.activities.base.BaseActivity;
+
+public class SplashActivity extends BaseActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        startActivity(new Intent(this, LoginActivity.class));
```

Comment:
Why always login, check here is passcode present or not like this https://github.com/openMF/mifos-mobile/blob/development/app/src/main/java/org/mifos/mobilebanking/ui/activities/LoginActivity.java#L62. If yes then open passcode activity otherwise open login activity.